Students/Naomi/ObjcTextInput/ObjcTextInput/Modules/TextInput/TIIUIViewController.m

```diff
@@ -0,0 +1,223 @@
+//
+//  TIIUIViewController.m
+//  ObjcTextInput
+//
+//  Created by Naomi De Leeuw on 09/11/2017.
+//  Copyright © 2017 iCapps. All rights reserved.
+//
+
+#import "TIIUIViewController.h"
+#import "TIILabelCollectionViewCell.h"
+#import "ObjcTextInput-Swift.h"
+
+@import Faro;
+
+@interface TIIUIViewController ()
+
+@property (weak, nonatomic) IBOutlet UICollectionView *labelCollectionView;
+@property (strong, nonatomic) NSMutableArray *names;
+@property (assign, nonatomic) NSInteger indexPathRow;
+@property (nonatomic, strong) PostService * service;
+@property (nonatomic, strong) id previewContext;
+@property (nonatomic) BOOL canEdit;
+@property (nonatomic, strong) TIITransitionManager *transitionManager;
+
+@end
+
+@implementation TIIUIViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    self.service = [[PostService alloc] init];
+    self.transitionManager = [[TIITransitionManager alloc] init];
+    self.canEdit = NO;
+
+    UIPasteConfiguration * config = [[UIPasteConfiguration alloc]init];
+    self.view.pasteConfiguration = config;
+
+    if([self isForceTouchAvailable]) {
+        self.previewContext = [self registerForPreviewingWithDelegate:self sourceView:self.labelCollectionView];
+    }
+
+    UILongPressGestureRecognizer *longPress = [[UILongPressGestureRecognizer alloc] initWithTarget:self action:@selector(handleLongPress:)];
+    [self.labelCollectionView addGestureRecognizer:longPress];
+
```

Comment:
Moet die long press niet in een else? lijkt me dat als er force touch is dat je geen long press wil toevoegen?